sky/packages/sky/lib/src/material/popup_menu.dart

```diff
@@ -159,12 +154,12 @@ class _MenuRoute extends PerformanceRoute {
   }
 }
 
-Future showMenu({ NavigatorState navigator, MenuPosition position, PopupMenuItemsBuilder builder, int level: 4 }) {
+Future showMenu({ BuildContext context, MenuPosition position, List<PopupMenuItem> items, int level: 4 }) {
   Completer completer = new Completer();
-  navigator.push(new _MenuRoute(
+  Navigator.of(context).push(new _MenuRoute(
     completer: completer,
     position: position,
-    builder: builder,
+    items: items,
```

Comment:
This will break the checkbox in the menu. That's what the work around for https://github.com/flutter/engine/issues/1528 was avoiding, but this will stop it from working.